docker/build-tools/Dockerfile

```diff
@@ -49,14 +49,14 @@ ENV GOLANG_GRPC_PROTOBUF_VERSION=v1.2.0
 ENV GOLANGCI_LINT_VERSION=v1.51.2
 ENV HADOLINT_VERSION=v2.12.0
 ENV HELM3_VERSION=v3.11.1
-ENV HUGO_VERSION=0.110.0
+ENV HUGO_VERSION=0.111.0
```

Comment:
I think 0.111.2 is the most recent. I know there is a PR to move up to that in istio.io.